### PR TITLE
RIA-1154 send decision and reasons

### PR DESCRIPTION
**Pull request by makowalski**
### JIRA link (if applicable) ###
[RIA-1154](https://tools.hmcts.net/jira/browse/RIA-1154)

### Change description ###
draft tests for send decision and reasons state
branch was created based on RIA-1152 & RIA-1153 branch

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/features/send-decision-and-reasons.feature [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    And I add the appeal response
+    And I request hearing requirements
+    And I list the case
+    And I create case summary
+    And I generate the hearing bundle
+    And I start decision and reasons
+    And I generate decision and reasons
+
+  @send-decision-and-reasons-dismissed @RIA-1154
+  Scenario: Send decision and reasons with Dismissed status
```
Comment (raghera) [inline e2e/features/send-decision-and-reasons.feature]: Is this not the scenario for Allowed?

File: e2e/features/send-decision-and-reasons.feature [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    When I click the `Overview` tab
+    And I click the `Send decision and reasons` link
+    Then I should see the text `Complete decision and reasons`
+    And I should see the text `What is your decision?`
+    And I should see the text `If the appeal is dismissed on all grounds, select 'Dismissed on all grounds'. If the appeal is allowed on any grounds, you must select 'Allowed'.`
+
+    When I click the `Allowed` label
```
Comment (raghera) [inline e2e/features/send-decision-and-reasons.feature]: Allowed

File: e2e/features/send-decision-and-reasons.feature [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    When I click the `Documents` tab
+    Then I should see the `Documents` page
+    And within the `Decision and reason documents` collection's first item, I should see `-Gonzlez-Decision-and-reasons-FINAL.pdf` in the `Document` field
+    And within the `Decision and reason documents` collection's first item, I should see `{$TODAY|D MMM YYYY}` for the `Date uploaded` field
+
+    And within the `Decision and reason documents` collection's second item, I should see `-Gonzlez-Decision-and-reasons-Cover-letter.PDF` in the `Document` field
+    And within the `Decision and reason documents` collection's second item, I should see `{$TODAY|D MMM YYYY}` for the `Date uploaded` field
+
+  @send-decision-and-reasons-allowed @RIA-1154
+  Scenario: Send decision and reasons with Allowed status
```
Comment (raghera) [inline e2e/features/send-decision-and-reasons.feature]: This scenario looks like the dismissed case.  Probably the scenario titles are wrong way around.

File: e2e/features/send-decision-and-reasons.feature (hunk 1 above)
Comment (makowalski) [reply e2e/features/send-decision-and-reasons.feature]: The first version contains checks for "Change" links in confirmation page where the decision was changed to dismissed.
So eventually Dismissed status was submitted for the case.

I think Tony removed that part, maybe it was not very clear what is going on.

File: e2e/features/send-decision-and-reasons.feature (hunk 3 above)
Comment (makowalski) [reply e2e/features/send-decision-and-reasons.feature]: The same situation here, I started with "Dismissed" and then checks for "Previous" button functionality changed the status to "Allowed", so the the case was submitted with "Allowed" status.

**Comment by makowalski** [comment]
I updated test titles + rebasing

**Pull request merged**